pyicloud/services/contacts.py

```diff
@@ -44,6 +45,19 @@ def refresh_client(self, from_dt=None, to_dt=None):
             params=params_contacts
         )
         self.response = req.json()
+        params_next = dict(self.params)
+        params_next.update({
+            'clientVersion': '2.1',
+            'prefToken': req.json()["prefToken"],
+            'syncToken': req.json()["syncToken"],
+            'limit': '0',
+            'offset': '0',
+        })
+        req = self.session.get(
```

Comment:
So there is 2 requests to get 1 result ?

Please do 2 requests if more than 499 contacts only **OR** 1 request to get 0 to infinite contacts at once.